.github/workflows/cd-common.yaml

```diff
@@ -1,4 +1,4 @@
-name: CD
+came: CD
```

Comment:
Probably incorrect 🤣

Reply:
lol. if I press <leader> in vim without a follow-up key, it inserts letter `c`. 

my config was like that for months now and I can't seem to find the culprit.